{"name": "test_login_usernameFormat_error", "status": "passed", "attachments": [{"name": "log", "source": "afc67a94-8d16-41bd-bb46-194ce50b2100-attachment.txt", "type": "text/plain"}], "start": 1610622934839, "stop": 1610622938611, "uuid": "c25935ef-5cb7-4c5a-80c2-280dfb7a55bf", "historyId": "b7c9963723357b0faf6d03130d678f90", "testCaseId": "e5bc1aeb101de12749be698a7235f84d", "fullName": "TestCases.test_logincase.test_login.Test_Login#test_login_usernameFormat_error", "labels": [{"name": "story", "value": "测试账号错误"}, {"name": "parentSuite", "value": "TestCases.test_logincase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_Login"}, {"name": "host", "value": "USER-20190821YP"}, {"name": "thread", "value": "8460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_logincase.test_login"}]}